tests/test_tasks.py

```diff
@@ -250,3 +250,38 @@ def test_receivers_exception(app, db, sample_record_string):
                 spider="Test",
                 workflow="test"
             )
+
+
+def test_create_workflow_for_faulty_data(app, db, halt_workflow):
+    """Test submit_results passing the data as payload."""
+    job_id = uuid.uuid4().hex  # init random value
+    with app.app_context():
+        CrawlerJob.create(
+            job_id=job_id,
+            spider="desy",
+            workflow=halt_workflow.__name__,
+            logs=None,
+            results=None,
+        )
+        db.session.commit()
+
+    with app.app_context():
+        job = CrawlerJob.get_by_job(job_id)
+        assert job
+        assert str(job.status)
```

Comment:
why ``str``?

Reply:
well this is some code that I mimiced for the other tests to make sure that the job is there. Maybe its testing that it can be stringified?